tools/make/kube.mk

```diff
@@ -40,6 +40,17 @@ endif
 YEAR := $(shell date +%Y)
 CONTROLLERGEN_OBJECT_FLAGS :=  object:headerFile="$(ROOT_DIR)/tools/boilerplate/boilerplate.generatego.txt",year=$(YEAR)
 
+.PHONY: prepare-ip-family
+prepare-ip-family: ## Prepare the environment for running e2e tests with ENVOY_PROXY_IP_FAMILY
+	@$(LOG_TARGET)
+ifeq ($(IP_FAMILY),ipv4)
```

Comment:
can we just reuse the EG enums in make file and scripts to reduce cognitive complexity ?

Reply:
`kind` use `ipv4` `ipv6` and `dual`, which is different with EG.

Reply:
ah cool